client/src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+module.exports = function (app) {
+  app.use(
+    "/api",
+    createProxyMiddleware({
+      target: process.env.REACT_APP_PROXY,
+      changeOrigin: true,
+    })
+  );
+};
```

Comment:
With this change, we can now have the proxy setup between any port setup in the .env file. This makes it so that we can remove the "proxy" setting in the client/package.json file. 

Useful links:
https://create-react-app.dev/docs/adding-custom-environment-variables/